round: [5,16]
rectangle: [2,14,18,20]
elliptical: [3,8,12,15,17]

object_list: [2,3,5,8,12,14,15,16,17,18,20]

